Yoast/wordpress-seo
# Add database logging

## Summary

When the current environment is defined as a development environment (through defining `YOAST_ENVIRONMENT` as `development`), this adds logging for both IdiORM and WPDB queries as well as memory usage. Sample output:

```html

```


This PR can be summarized in the following changelog entry:

* None needed.

## Relevant technical choices:

* Created a new class for this that is only instantiated on development setups.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have this in your `wp-config.php`:
  ```php
  define( 'YOAST_ENVIRONMENT', 'development' );
  ```
* See the source of any page to see output like the above in the comments at the end of the page.

## Quality assurance
